sormas-ui/src/main/java/de/symeda/sormas/ui/labmessage/LabMessageController.java

```diff
@@ -675,6 +675,10 @@ private void createPathogenTest(SampleDto sampleDto, LabMessageDto labMessageDto
 		Window window = VaadinUiUtil.createPopupWindow();
 		CommitDiscardWrapperComponent<PathogenTestForm> pathogenTestCreateComponent =
 			getPathogenTestCreateComponent(sampleDto, labMessageDto, pathogenTestDto, window);
+		CheckBox viaLimsCheckbox = pathogenTestCreateComponent.getWrappedComponent().getField(PathogenTestDto.VIA_LIMS);
+		viaLimsCheckbox.setValue(Boolean.TRUE);
```

Comment:
[Here](https://github.com/hzi-braunschweig/SORMAS-Project/blob/9ea12de565dace01f2544b396be2acacf69c0535/sormas-ui/src/main/java/de/symeda/sormas/ui/labmessage/LabMessageController.java#L652) is a null check involved. Is there a reason why you ommited this check in your implementation?

Reply:
The PathogenTestForm contains that field. I saw no reason to check for null.

Reply:
Does it always exist, also for non german systems?

Reply:
No, it does not.

Reply:
After a deeper look it seems that it always exists. I am wondering what the null check is there for...